### PR TITLE
Volunteer signup

### PR DESCRIPTION
**Pull request by sohak22**
Notion ticket link:

https://www.notion.so/uwblueprintexecs/FRONTEND-Volunteer-Participant-Info-Sign-Up-Page-2efff449c0aa4819a3abbd4a5b936d6d

### Implementation description

- Front end volunteer sign up form, with capabilities to save fields in local storage

### Steps to test

1. localhost:3000/volunteersignup

### Checklist

-   [ ] My PR name is descriptive and in imperative tense
-   [ ] I have run the linter
-   [ ] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: pages/volunteersignup.tsx
Comment (mahadzaryab1) [inline pages/volunteersignup.tsx]: flagging this TODO

File: pages/volunteersignup.tsx
Comment (mahadzaryab1) [inline pages/volunteersignup.tsx]: what exactly is the session being used for in this page?

File: pages/volunteersignup.tsx
Comment (mahadzaryab1) [inline pages/volunteersignup.tsx]: colors can be moved to `src/styles`

**Review by mahadzaryab1 — COMMENTED** [review]
Added a couple of comments/questions. Also let's rename the page to something else - `volunteers/signup` maybe?

File: pages/volunteersignup.tsx
Comment (sohak22) [reply pages/volunteersignup.tsx]: The session is for the wrapper

File: pages/volunteersignup.tsx
Comment (jason17h) [inline pages/volunteersignup.tsx]: can you maybe change this to `heardFrom` or something more descriptive

File: pages/volunteersignup.tsx
Comment (jason17h) [inline pages/volunteersignup.tsx]: what's this / is it needed

File: pages/volunteersignup.tsx
Comment (jason17h) [inline pages/volunteersignup.tsx]: same as above

File: pages/volunteersignup.tsx
Comment (jason17h) [inline pages/volunteersignup.tsx]: doesn't look like this is used anywhere yet

**Review by jason17h — COMMENTED** [review]
Looking good so far, just a few comments + I added some changes so be sure to pull!!

File: pages/volunteersignup.tsx
Comment (jason17h) [inline pages/volunteersignup.tsx]: looks like `RADIO_YES` and `RADIO_NO` are not used anymore so they can be removed. same with some of the imports I think

File: pages/volunteersignup.tsx
Comment (jason17h) [inline pages/volunteersignup.tsx]: these don't look like they're used - should they be used somewhere? or can we get rid of them? it looks like they aren't needed

**Review by jason17h — APPROVED** [review]
just some minor comments to remove unused things but other than that looks good to me (should get at least 1 more approval though)! Thanks for doing this 😎

File: pages/volunteersignup.tsx
Comment (sohak22) [reply pages/volunteersignup.tsx]: Wait im not sure which one your talking about but Skills is used on line 238-239, heard from is used under that on lines 249-250. And I took out attending

File: src/style/colours.ts
Comment (jason17h) [inline src/style/colours.ts]: can you move this to `src/styles` instead of `src/style`

File: pages/volunteersignup.tsx
Comment (jason17h) [inline pages/volunteersignup.tsx]: build is failing because there is no `LightBlue` defined

File: pages/volunteersignup.tsx
Comment (jason17h) [reply pages/volunteersignup.tsx]: I was just referring to attending

**Review by jason17h — APPROVED** [review]

**Review by feng-eric — APPROVED** [review]

**Comment by jason17h** [comment]
Btw @sohak22 can you update the signup page to redirect to this form if you click the "Volunteer" button? https://github.com/uwblueprint/social-diversity-for-children/blob/71edf4e5961fd4d76680b4f998e45bb78eae3da8/pages/signup.tsx#L54

**Review by feng-eric — COMMENTED** [review]
Sorry, thought I submitted the review a few days ago. Had one small change I wanted to comment on.

**Review by feng-eric — APPROVED** [review]
LGTM, ping me before you merge because we need to reset the database since we're making DB schema changes.

**Pull request merged**
